--- configure.in.orig	2010-01-01 14:29:57.000000000 +0100
+++ configure.in	2010-01-01 14:30:11.000000000 +0100
@@ -161,11 +161,7 @@
 AC_SUBST(HAVE_DOT)
 
 # Similar for latex
-HAVE_LATEX=YES
-AC_PATH_PROG(LATEX, latex, )
-if test x$LATEX = x; then
-	HAVE_LATEX=NO
-fi
+HAVE_LATEX=NO
 # This variable is used to turn LATEX on and off in the Doxyfile
 AC_SUBST(HAVE_LATEX)
 # This variable is used in doc/Makefile.am to set the target to make
