diff -Nur log4c-1.0.10.orig/doc/Makefile.am log4c-1.0.10.chng/doc/Makefile.am
--- log4c-1.0.10.orig/doc/Makefile.am	Wed Jul 30 10:56:20 2003
+++ log4c-1.0.10.chng/doc/Makefile.am	Wed Jul 30 10:57:12 2003
@@ -25,7 +25,7 @@
 		echo "$(INSTALL_DATA) $$i $(DESTDIR)$(docdir)"; \
 		$(INSTALL_DATA) $$i $(DESTDIR)$(docdir); \
 	done
-	cp -r html $(docdir)
+	cp -r html $(DESTDIR)$(docdir)
 
 uninstall-local:
 	@for i in ./man/man3/*.3; do \
@@ -33,7 +33,7 @@
 		echo "rm -f $(DESTDIR)$(man3dir)/$$inst"; \
 		rm -f $(DESTDIR)$(man3dir)/$$inst; \
 	done
-	rm -rf $(docdir)/*
+	rm -rf $(DESTDIR)$(docdir)/*
 
 clean-local:
 	$(RM) -r latex man html doxygen.log
